projects/angular-email/tailwindcss-preset-email/mso.css

```
@@ -0,0 +1,433 @@
+  mso-border-right-source: --value("auto", "background", 'foreground');
+}
+@utility mso-border-top-source-* {
+  mso-border-top-source: --value("auto", "background", 'foreground');
+}
+
+@utility mso-border-shadow-* {
+  mso-border-shadow: --value("yes", "no");
+}
+
+@utility mso-border-effect-* {
+  mso-border-shadow: --value("3d", "box");
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)